### ISSUE TITLE
Repartition topic is created with 1 replica and min.insync.replica of 2

### ISSUE THREAD

**Issue by marghidanu** [faust-streaming/faust#390]
## Checklist

- [x] I have included information about relevant versions
- [ ] I have verified that the issue persists when using the `master` branch of Faust.

## Expected behavior

The repartition topic seems to be created with 1 replica and `min.insync.replicas` of 2. This causes the application to fail, although this is logged as a warning. 

## Actual behavior

Not to fail.

## Full traceback

```pytb
[WARNING] Got error produce response on topic-partition TopicPartition(topic='example.agents.count_shares-shares-Share.id-repartition', partition=6), retrying. Error: <class 'kafka.errors.NotEnoughReplicasError'> 
```

# Versions

* Python version: 3.9
* Faust version: 0.9.0
* Operating system: Debian
* Kafka version: 3.2.0 (AWS MSK)
* RocksDB version (if applicable)

**Comment by wbarnha** [comment]
Out of curiosity, does this still occur with `faust-streaming==0.8.11`?

**Comment by wbarnha** [comment]
After reviewing changes in https://github.com/faust-streaming/faust/pull/375, it's entirely possible we made a change that breaks things.

**Comment by wbarnha** [comment]
Wrong button, sorry.

**Comment by wbarnha** [comment]
Do you have two brokers running? I'm not sure how to recreate your issue, but it seems https://stackoverflow.com/questions/62346727/kafka-troubleshooting-notenoughreplicasexception is similar to what you're experiencing.

**Comment by marghidanu** [comment]
No, I currently have three brokers running. Is there a way to set the replicas property for these topics created by faust?

**Comment by wbarnha** [comment]
> No, I currently have three brokers running. Is there a way to set the replicas property for these topics created by faust?

Not directly, most of this is a product of https://aiokafka.readthedocs.io/en/stable/producer.html#retries-and-message-acknowledgement. Under https://faust-streaming.github.io/faust/userguide/settings.html#id51 you could alter the ack settings to change the behavior of `aiokafka`.

Three brokers should be fine... are you passing all three brokers in your `broker` argument to `faust.App`?

**Comment by marghidanu** [comment]
Yes, everything si configured correctly, I'm also using authentication.

**Comment by wbarnha** [comment]
Try changing your value in `app.conf` for https://faust-streaming.github.io/faust/userguide/settings.html#topic-replication-factor from the default value of 1 to 3, and let me know what you get.

**Comment by marghidanu** [comment]
That seems to work! Now The application is working as expected.

**Comment by wbarnha** [comment]
Awesome, glad I could help!
